### ISSUE TITLE
Fixing n+1 queries (eager load attachinary associations)

### ISSUE THREAD

**Issue by jfeust** [assembler/attachinary#113]
When displaying a list of records that need to display attachinary images, I see no way around the n+1 query on each Attachinary::File.

Is there any way to resolve this?

**Comment by orlando** [comment]
You can eager load attachinary. For example, I have `Photo` and `Avatar` models

```ruby
# model.rb
    has_attachment :avatar
    has_attachments :photos, maximum: 10

# controller.rb
Model.includes(:avatar_files, :photos_files)
```

**Comment by orlando** [comment]
I think we should add this information to the README

**Comment by jfeust** [comment]
@orlando Very cool, thanks.  +1 For the README.

**Comment by nunommc** [comment]
Did you add it?

**Comment by orlando** [comment]
@nunommc thanks for the remainder, I just merged #122 adding that information
